osidb/migrations/0143_rename_flaw_fields.py

```diff
@@ -6,7 +6,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('osidb', '0141_remove_flawacknowledgment_meta_attr'),
+        ('osidb', '0142_flaw_major_incident_start_dt'),
```

Comment:
I do not think that it is very safe to mess with the existing migrations. They are most probably already applied somewhere which is recorded in the DB etc. I would expect this to result in some nasty Django errors. You should rather rename the new validation.

Reply:
Oops, this is actually a mistake from when I rebased